packages/big-design/src/components/Toggle/styled.ts

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+
+export const StyledButton = styled.button<{ isActive: boolean }>`
+  align-items: center;
+  appearance: none;
+  background-color: ${({ isActive, theme }) =>
+    isActive ? theme.colors.primary20 : theme.colors.white};
+  border: 1px solid
+    ${({ theme, isActive }) => (isActive ? theme.colors.primary30 : theme.colors.secondary30)};
+  border-left-width: 0.5px;
+  border-right-width: 0.5px;
```

Comment:
We can face an issue when the border will be shown as `1px` instead of `0.5px` on screens with low PPI(pixels per inch).

Is it possible to use something like in the example below or some equal technique:

```
border: 1px solid
    ${({ theme, isActive }) => (isActive ? theme.colors.primary30 : theme.colors.secondary30)};
border-left-width: 0;

&:first-child {
  border-left-width: 1px;
}
```

Reply:
If you need that focused button will have all visible borders you can follow the existing code here https://github.com/bigcommerce/big-design/blob/master/packages/big-design/src/components/ButtonGroup/styled.tsx with the negative margins.

Reply:
@golcinho Thanks for the reference, Done!!!
😊